### PR TITLE
deprecate baseTemplates to simplify loading and OS list flicker

### PR DESCRIPTION
**Pull request by suomiy**
- disable fields when loading

- bug: https://bugzilla.redhat.com/show_bug.cgi?id=1730390

@mareklibra @rawagner 

IMO `web-ui-components` `base mock templates` shouldn't be needed anymore. Current usage causes inconsistent behaviour on user's side.

Thoughts?

### CODE REVIEW

**Comment by rawagner** [comment]
+1 for removing mocked templates

**Comment by mareklibra** [comment]
I agree that mock templates can be removed now, as the kubevirt's deployment has already settled down and templates are mandatory part of it. In addition, the dialog won't be rendered without kubevirt installed.

File: src/components/Wizard/CreateVmWizard/CreateVmWizard.js [hunk 1]
```diff
@@ -329,6 +330,7 @@ CreateVmWizard.propTypes = {
   k8sPatch: PropTypes.func.isRequired,
   k8sKill: PropTypes.func.isRequired,
   onHide: PropTypes.func.isRequired,
+  templatesLoaded: PropTypes.bool,
```
Comment (mareklibra) [inline src/components/Wizard/CreateVmWizard/CreateVmWizard.js]: I left comment regarding this prop in https://github.com/openshift/console/pull/2157

Comment (mareklibra) [reply src/components/Wizard/CreateVmWizard/CreateVmWizard.js]: Agreed in 2157

**Pull request merged**